### PR TITLE
Customer Home Styles Refactoring

### PR DESCRIPTION
**Pull request by davemart-in**
## Changes proposed in this Pull Request

There were a number of items that were visually off when compared to the [HTML prototype](https://codepen.io/lessbloat/pen/PMEwXE?editors=1100). Changes include:

- Refactored CSS in customer-home/styles.scss in https://github.com/Automattic/wp-calypso/commit/b7bbf154ba466a675dd53b23ab78a77350b224ef 
- Fixed layout column spacing in https://github.com/Automattic/wp-calypso/commit/4682a32a883f52ea1fb5f3d6b64e5fcc24a3d456
- Tightened up styles in left column in https://github.com/Automattic/wp-calypso/commit/21c0257e5fc4881b1d84fdc624274db7efca22f3
- Tightened up styles in right column in https://github.com/Automattic/wp-calypso/commit/7a90023d95939fa79395bcd406abd20164467556

## Preview

| Before  | After |
| ------------- | ------------- |
| before | after |

### CODE REVIEW

File: client/my-sites/customer-home/style.scss [hunk 1]
```diff
@@ -1,110 +1,174 @@
...
+		margin-bottom: 16px;
+		width: 100%;
+
+		.button {
+			border-width: 1px;
+			display: block;
+			min-height: 85px;
+			min-width: 100%;
+			padding: 24px;
+	    text-align: center;
```
Comment (scruffian) [inline client/my-sites/customer-home/style.scss]: Indentation off here

File: client/my-sites/customer-home/style.scss [hunk 2]
```diff
@@ -1,110 +1,174 @@
...
+			margin-bottom: 24px;
+			width: 50%;
 
-.customer-home__layout {
-	@include breakpoint('>960px') {
+			.button {
+		    margin: 0 12px;
+		    min-height: 105px;
+		    min-width: 90%;
+		    padding: 30px 24px;
```
Comment (scruffian) [inline client/my-sites/customer-home/style.scss]: more indentation issues

File: client/my-sites/customer-home/style.scss (hunk 1 above)
Comment (davemart-in) [reply client/my-sites/customer-home/style.scss]: Good eye! Fixed in: https://github.com/Automattic/wp-calypso/commit/81e150b3376bee4572ded0f896b7bdf06b424fe0

File: client/my-sites/customer-home/style.scss (hunk 2 above)
Comment (davemart-in) [reply client/my-sites/customer-home/style.scss]: Thanks. Fixed in: https://github.com/Automattic/wp-calypso/commit/81e150b3376bee4572ded0f896b7bdf06b424fe0

File: client/my-sites/customer-home/style.scss [hunk 3]
```diff
@@ -1,110 +1,174 @@
...
+				margin-right: 12px;
 			}
 		}
 	}
+	&__card-col-right {
+		width: 100%;
 
-	span {
-		display: block;
+		@include breakpoint('>1040px') {
```
[the remark this answers is not in the document]
Comment (davemart-in) [reply client/my-sites/customer-home/style.scss]: Yep, they were mostly duplicated. Good call. I removed the duplication in https://github.com/Automattic/wp-calypso/commit/7529bc900df076ec72a34b021527e8c6c06c6c9d

**Comment by scruffian** [comment]
This looks good to me. There's one breakpoint that still looks kind of odd:

Screenshot 2019-10-22 at 17 34 06

compared to: 

Screenshot 2019-10-22 at 17 33 56

This shouldn't stop us merging this PR thought

**Comment by davemart-in** [comment]
> There's one breakpoint that still looks kind of odd

@scruffian, thanks for highlighting that! Fixed in https://github.com/Automattic/wp-calypso/commit/9499c44d3a0b1bda9d6d93f1590d8d45a80be4b6

**Preview**

preview

**Pull request merged**